src/components/instances/new/MetaLocal.js

```diff
@@ -50,7 +51,20 @@ function MetaLocal() {
 				setFormState({ error: 'usernames must have only letters, underscores, and hyphens' });
 			} else if (instance_name.length && user.length && pass.length && host.length && port.length) {
 				try {
-					const currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
+					let currentUser;
+					if (pass) {
```

Comment:
I still don't understand the purpose of this whole code block/change, why login if the password is already persisted? What is wrong with the previous one-liner to get userInfo?

Reply:
I reverted it